.scrutinizer.yml

```diff
@@ -0,0 +1,12 @@
+inherit: true
+
+tools:
+    external_code_coverage: true
```

Comment:
looks like some helpful robot beat me to the punch (card): https://github.com/silverstripe/silverstripe-translatable/pull/204